src/pages/login/Login.jsx

```diff
@@ -1,57 +1,109 @@
-import React from 'react';
-
-import { Typography, Card, Button, TextField, Stack, Box } from '@mui/material';
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import {
+  Typography,
+  Card,
+  Button,
+  TextField,
+  Stack,
+  Box,
+  FormControl,
+} from '@mui/material';
+import { userLogIn } from '../../store/actions';
 
 import './Login.sass';
-
 import { Footer } from '../../components';
 import { useMediaDown } from '../../components/utils';
 import { getFieldLabel } from '../../utils';
 
-const Login = () => {
+const Login = (props) => {
+  const { message } = props;
   const smallScreen = useMediaDown('sm');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleChange = (event) => {
+    const { name, value } = event.target;
```

Comment:
you can manage email and password in one state object and handle change in a smaller function. this is only a suggestion for refactoring